pkg/app/piped/cloudprovider/cloudrun/cloudrun.go

```diff
@@ -126,6 +126,26 @@ func (s *Service) ServiceManifest() (ServiceManifest, error) {
 	return ParseServiceManifest(data)
 }
 
+func (s *Service) UID() string {
+	if s.Metadata == nil {
+		return ""
+	}
+	return s.Metadata.Uid
+}
+
+// RevisionNames return some revision names which may handle traffic.
+func (s *Service) RevisionNames() []string {
```

Comment:
How about naming it with "Active" to narrow down the list?

```suggestion
func (s *Service) ActiveRevisionNames() []string {
```